features/number.yml

```diff
@@ -0,0 +1,85 @@
+compat_features:
+  - javascript.builtins.Infinity
+  - javascript.builtins.isNaN
+  - javascript.builtins.Math
+  - javascript.builtins.Math.abs
+  - javascript.builtins.Math.acos
+  - javascript.builtins.Math.acosh
+  - javascript.builtins.Math.asin
+  - javascript.builtins.Math.asinh
+  - javascript.builtins.Math.atan
+  - javascript.builtins.Math.atan2
+  - javascript.builtins.Math.atanh
+  - javascript.builtins.Math.cbrt
+  - javascript.builtins.Math.ceil
+  - javascript.builtins.Math.clz32
+  - javascript.builtins.Math.cos
+  - javascript.builtins.Math.cosh
+  - javascript.builtins.Math.E
+  - javascript.builtins.Math.exp
+  - javascript.builtins.Math.expm1
+  - javascript.builtins.Math.f16round
+  - javascript.builtins.Math.floor
+  - javascript.builtins.Math.fround
+  - javascript.builtins.Math.hypot
+  - javascript.builtins.Math.imul
+  - javascript.builtins.Math.LN10
+  - javascript.builtins.Math.LN2
+  - javascript.builtins.Math.log
+  - javascript.builtins.Math.log10
+  - javascript.builtins.Math.LOG10E
+  - javascript.builtins.Math.log1p
+  - javascript.builtins.Math.log2
+  - javascript.builtins.Math.LOG2E
+  - javascript.builtins.Math.max
+  - javascript.builtins.Math.min
+  - javascript.builtins.Math.PI
+  - javascript.builtins.Math.pow
+  - javascript.builtins.Math.random
+  - javascript.builtins.Math.round
+  - javascript.builtins.Math.sign
+  - javascript.builtins.Math.sin
+  - javascript.builtins.Math.sinh
+  - javascript.builtins.Math.sqrt
+  - javascript.builtins.Math.SQRT1_2
+  - javascript.builtins.Math.SQRT2
+  - javascript.builtins.Math.tan
+  - javascript.builtins.Math.tanh
+  - javascript.builtins.Math.trunc
+  - javascript.builtins.NaN
+  - javascript.builtins.Number
+  - javascript.builtins.Number.EPSILON
+  - javascript.builtins.Number.isFinite
+  - javascript.builtins.Number.isInteger
+  - javascript.builtins.Number.isNaN
+  - javascript.builtins.Number.isSafeInteger
+  - javascript.builtins.Number.MAX_SAFE_INTEGER
+  - javascript.builtins.Number.MAX_VALUE
+  - javascript.builtins.Number.MIN_SAFE_INTEGER
+  - javascript.builtins.Number.MIN_VALUE
+  - javascript.builtins.Number.NaN
+  - javascript.builtins.Number.NEGATIVE_INFINITY
+  - javascript.builtins.Number.Number
+  - javascript.builtins.Number.parseFloat
+  - javascript.builtins.Number.parseInt
+  - javascript.builtins.Number.POSITIVE_INFINITY
+  - javascript.builtins.Number.toExponential
+  - javascript.builtins.Number.toFixed
+  - javascript.builtins.Number.toLocaleString
+  - javascript.builtins.Number.toLocaleString.locales_parameter
+  - javascript.builtins.Number.toLocaleString.options_parameter
+  - javascript.builtins.Number.toPrecision
+  - javascript.builtins.Number.toString
+  - javascript.builtins.Number.valueOf
+  - javascript.builtins.parseInt
+  - javascript.grammar.binary_numeric_literals
+  - javascript.grammar.decimal_numeric_literals
+  - javascript.grammar.hexadecimal_numeric_literals
+  - javascript.grammar.numeric_separators
```

Comment:
Let's do as @Elchi3 suggests and break this into its own feature. A quick search reveals lots of blog posts and other commentary on this, justifying its own feature.

Reply:
done